### ISSUE TITLE
noVNC can't connect to high resolution in MAC 

### ISSUE THREAD

**Issue by keksanket** [novnc/noVNC#811]
Hi ,
I am working on macbook PRO retina , Operating System - Sierra . When I am trying to connect to my own local machine(Which is on  a high resolution) using websockify , it shows connected and then it abruptly disconnects.

Console output recieved onclose callback is  :

close { target: WebSocket, isTrusted: true, wasClean: false, code: 1006, reason: "", currentTarget: WebSocket, eventPhase: 2, bubbles: false, cancelable: false, defaultPrevented: false, composed: false }

so basically it is sending 1006 code without any reason. 

Now , when I reduce the resolution of my macbook. It connects without throwing any error .

How can I debug this issue?

Thanks

**Comment by samhed** [comment]
What does the websockify output say?

**Comment by CendioOssman** [comment]
Unfortunately the browsers are a bit shit at giving diagnostics for websockets. But the first step should be to check the server for logs, so you can see if this is a network issue or a local browser problem.

You didn't write which browser you were using, so I'm assuming it is Safari? Do you see the same issues with Firefox or Chrome?

**Comment by CendioOssman** [comment]
Also, Apple's remote desktop server isn't really proper VNC, so they might have introduced new problems when in retina mode. I suggest testing a connection to some other VNC server to see if it's a client or server issue.

**Comment by keksanket** [comment]
@CendioOssman  actually this problem is persistent with all the browsers that I am using. It is the same for chrome and mozilla as well. 
@samhed  I will revert back with websockify output.

Thanks

**Comment by DirectXMan12** [comment]
might be something weird in the retina mode protocol :-/.  We'd need a recording from websockify to debug further.

**Comment by samhed** [comment]
@keksanket you think it would be possible to provide us with a recording?

**Comment by samhed** [comment]
Closing due to no response.
